### PR TITLE
Fetch Json-LD definitions from local ressources

### LINKED ISSUE #10254: messages do not arrive reliably
- [x] I have searched open and closed issues for duplicates
May be related to #9544 and/or #6395

### Bug Description
My instance does not seem to get new messages over ActivityPub reliably. E.g. a test message explizitly adressed to my user from nerdculture.de does not arrive on my instance.

### Steps to Reproduce
???

### Log
the log shows a lot of the following errors:
```
2021-05-16T10:44:16Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"0ef562","process_id":1340230}
2021-05-16T10:44:39Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"469feb","process_id":1341924}
2021-05-16T10:44:44Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"9e3d45","process_id":1341924}
2021-05-16T10:44:52Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"7d7409","process_id":1341924}
2021-05-16T10:44:57Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"9f7ec7","process_id":1341924}
2021-05-16T10:45:02Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"0437bf","process_id":1341924}
2021-05-16T10:45:05Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"1ad88c","process_id":1341924}
2021-05-16T10:45:39Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"14d7bc","process_id":1347908}
2021-05-16T10:45:44Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"309100","process_id":1347908}
2021-05-16T10:46:00Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"e70a50","process_id":1348794}
2021-05-16T10:46:04Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"00fb38","process_id":1348794}
2021-05-16T10:46:14Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"75d218","process_id":1348794}
2021-05-16T10:46:35Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"d2207f","process_id":1348794}
2021-05-16T10:46:40Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"9acc21","process_id":1351120}
2021-05-16T10:46:42Z index [ERROR]: compacting error [] - {"file":"JsonLD.php","line":153,"function":"compact","uid":"cf5023","process_id":1348794}
```

### Actual Result:
Messages do not arrive, but sometimes they get through.

### Expected Result:
Messages arrive reliably 

### Platform Info
Friendica Version: 2021.04
Friendica Source: git/stable
PHP version: 7.4.14
PHP memory limit: 256MB
SQL version: MariaDB 10.5

### PR DESCRIPTION
**Pull request by annando**
This is a follow up fix for the defunct URL `https://w3id.org/security/v1`. we now don't use compacted data when fetching account data.

I tested against some accounts from several systems - this seemed to work. But it could happen that some exotic systems send data in a format that we now can't parse anymore. We have to see this in the future.

### CODE REVIEW

File: src/Model/APContact.php [hunk 1]
```diff
@@ -243,43 +241,43 @@ public static function getByURL($url, $update = null)
...
-			$data = ActivityPub::fetchContent($apcontact['following']);
-			if (!empty($data)) {
-				if (!empty($data['totalItems'])) {
-					$apcontact['following_count'] = $data['totalItems'];
+			$content = ActivityPub::fetchContent($apcontact['following']);
+			if (!empty($content)) {
+				if (!empty($content['totalItems'])) {
+					$apcontact['following_count'] = $content['totalItems'];
 				}
 			}
```
Comment (MrPetovan) [inline src/Model/APContact.php]:
```suggestion
			$following = ActivityPub::fetchContent($apcontact['following']);
			if (!empty($following['totalItems'])) {
				$apcontact['following_count'] = $following['totalItems'];
			}
```

File: src/Model/APContact.php [hunk 2]
```diff
@@ -243,43 +241,43 @@ public static function getByURL($url, $update = null)
...
-			$data = ActivityPub::fetchContent($apcontact['followers']);
-			if (!empty($data)) {
-				if (!empty($data['totalItems'])) {
-					$apcontact['followers_count'] = $data['totalItems'];
+			$content = ActivityPub::fetchContent($apcontact['followers']);
+			if (!empty($content)) {
+				if (!empty($content['totalItems'])) {
+					$apcontact['followers_count'] = $content['totalItems'];
 				}
 			}
```
Comment (MrPetovan) [inline src/Model/APContact.php]:
```suggestion
			$followers = ActivityPub::fetchContent($apcontact['followers']);
			if (!empty($followers['totalItems'])) {
				$apcontact['followers_count'] = $followers['totalItems'];
			}
```

File: src/Model/APContact.php [hunk 3]
```diff
@@ -243,43 +241,43 @@ public static function getByURL($url, $update = null)
...
-			$data = ActivityPub::fetchContent($apcontact['outbox']);
-			if (!empty($data)) {
-				if (!empty($data['totalItems'])) {
-					$apcontact['statuses_count'] = $data['totalItems'];
+			$content = ActivityPub::fetchContent($apcontact['outbox']);
+			if (!empty($content)) {
+				if (!empty($content['totalItems'])) {
+					$apcontact['statuses_count'] = $content['totalItems'];
 				}
 			}
```
Comment (MrPetovan) [inline src/Model/APContact.php]:
```suggestion
			$outbox = ActivityPub::fetchContent($apcontact['outbox']);
			if (!empty($outbox['totalItems'])) {
				$apcontact['statuses_count'] = $outbox['totalItems'];
			}
```

**Review by MrPetovan — APPROVED** [review]

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Please remove the duplicated conditions and rename the variables.

**Review by MrPetovan — APPROVED** [review]

**Comment by annando** [comment]
@MrPetovan I'm still fighting with the issue. Although the messages are accepted again, the signature check doesn't work with all senders.

File: static/w3id-security-v1.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      "publicKeyBase58": "sec:publicKeyBase58",
+      "publicKeyPem": "sec:publicKeyPem",
+      "publicKeyService": {"@id": "sec:publicKeyService", "@type": "@id"},
+      "revoked": {"@id": "sec:revoked", "@type": "xsd:dateTime"},
+      "salt": "sec:salt",
+      "signature": "sec:signature",
+      "signatureAlgorithm": "sec:signingAlgorithm",
+      "signatureValue": "sec:signatureValue"
+    }
+  }
```
Comment (MrPetovan) [inline static/w3id-security-v1.json]: Coding standards: Please add an EOF EOL.
```suggestion
}

```

File: static/w3id-security-v1.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
+{
+    "@context": {
+      "id": "@id",
```
Comment (MrPetovan) [inline static/w3id-security-v1.json]: Code standards: Inconsistent indentation, please choose 2 or 4-spaces indentation for the whole file.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by annando** [comment]
I reverted most of my yesterday's panic coding. It is replaced with a loader that fetches the definitions locally if they are known.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**